test/hpr_protocol_router_SUITE.erl

```diff
@@ -304,19 +302,31 @@ grpc_full_flow_connection_refused_test(_Config) ->
 %% ===================================================================
 
 test_packet() ->
-    {packet_router_packet_up_v1_pb,
+    {
```

Comment:
Should this use the record syntax instead of constructing the tuple directly?

Reply:
I am fixing in another PR